src/sipm_simple_calibration.jl

```diff
@@ -0,0 +1,215 @@
+"""
+    sipm_simple_calibration(pe_uncal::Array)
+
+
+Perform a simple calibration for the uncalibrated p.e. spectrum array `pe_uncal`
+using just the 1 p.e. and 2 p.e. peak positions estimated by a peakfinder.
+
+Returns 
+    * `pe_simple_cal`: array of the calibrated pe array with the simple calibration
+    * `func`: function to use for the calibration
+    * `c`: calibration factor
+    * `peakpos`: 1 p.e. and 2 p.e. peak positions in uncalibrated amplitude
+    * `h_uncal`: histogram of the uncalibrated pe array
+    * `h_calsimple`: histogram of the calibrated pe array with the simple calibration
+"""
+
+function sipm_simple_calibration end
+export sipm_simple_calibration
+
+function sipm_simple_calibration(pe_uncal::Vector{<:Real}; 
+                                expect_noise_peak::Bool=false, 
+                                kwargs...)
+    
+    if expect_noise_peak
+        h_uncal, peakpos = find_peaks_noise_peak_exists(pe_uncal; kwargs...)
+    else
+        h_uncal, peakpos = find_peaks(pe_uncal; kwargs...)
+    end
+
+    # simple calibration
+    gain = peakpos[2] - peakpos[1]
+    c = 1/gain
+    offset = - (peakpos[1] * c - 1) 
+
+    f_simple_calib = Base.Fix1(*, c)
+    pe_simple_cal = pe_uncal .* c .+ offset
+
+    h_calsimple = histogram(pe_simple_cal, bins=0.5:.01:4.5)
+
+    result = (
+        pe_simple_cal = pe_simple_cal,
+        func = f_simple_calib, 
+        c = c,
+        offset = offset
+    )
+    report = (
+        peakpos = peakpos,
```

Comment:
Are the `peakpos` in ADC? Maybe also the the `peakpos` in `pe` should be given here for the plot.

Reply:
that doesn't make sense. the function is designed such that the calibrated peakpos are exactly at 1.0 and 2.0...

Reply:
Yes but for the plotting it helps. Because then you can say plot vertical lines at these positions for the calibrated values and at these positions for the uncalibrated values.